crypto-ffi/bindings/jvm/src/main/kotlin/com/wire/crypto/CoreCryptoException.kt

```diff
@@ -0,0 +1,346 @@
+//         *
+//         * Note that any [CryptoSession]s which might have been involved in a computation leading to
+//         * a panic must no longer be used as their in-memory state may be corrupt. Such sessions
+//         * should be closed and may be subsequently reloaded to retry the operation(s).
+//         */
+//        PANIC,
+//
+//        /** An unspecified error occurred. */
+//        UNKNOWN_ERROR,
+//
+//        /** Local files were not found. */
+//        LOCAL_FILES_NOT_FOUND,
+//    }
+//
+//    companion object {
+//        @Suppress("MagicNumber")
+//        fun fromNativeCode(code: Int): Code {
+//            return when (code) {
+//                1 -> Code.STORAGE_ERROR
+//                2 -> Code.SESSION_NOT_FOUND
+//                3 -> Code.DECODE_ERROR
+//                4 -> Code.REMOTE_IDENTITY_CHANGED
+//                5 -> Code.INVALID_SIGNATURE
+//                6 -> Code.INVALID_MESSAGE
+//                7 -> Code.DUPLICATE_MESSAGE
+//                8 -> Code.TOO_DISTANT_FUTURE
+//                9 -> Code.OUTDATED_MESSAGE
+//                13 -> Code.IDENTITY_ERROR
+//                14 -> Code.PREKEY_NOT_FOUND
+//                15 -> Code.PANIC
+//                else -> Code.UNKNOWN_ERROR
+//            }
+//        }
+//
+//        // Mapping source:
+//        // https://github.com/wireapp/proteus/blob/2.x/crates/proteus-traits/src/lib.rs
+//        // https://github.com/wireapp/wire-web-core/blob/7383e108f5e9d15d0b82c41ed504964667463cfc/packages/proteus/README.md
+//        fun fromProteusCode(code: UShort): Code {
+//            @Suppress("MagicNumber")
+//            return when (code.toInt()) {
+//                501 -> Code.STORAGE_ERROR
+//                102 -> Code.SESSION_NOT_FOUND
+//                3,
+//                301,
+//                302,
+//                303 -> Code.DECODE_ERROR
+//                204 -> Code.REMOTE_IDENTITY_CHANGED
+//                206,
+//                207,
+//                210 -> Code.INVALID_SIGNATURE
+//                200,
+//                201,
+//                202,
+//                205,
+//                213 -> Code.INVALID_MESSAGE
+//                209 -> Code.DUPLICATE_MESSAGE
+//                211,
+//                212 -> Code.TOO_DISTANT_FUTURE
+//                208 -> Code.OUTDATED_MESSAGE
+//                300 -> Code.IDENTITY_ERROR
+//                101 -> Code.PREKEY_NOT_FOUND
+//                5 -> Code.PANIC
+//                else -> Code.UNKNOWN_ERROR
+//            }
+//        }
+//
+//        fun fromCoreCryptoException(proteusLastErrorCode: UShort?, err: CoreCryptoException): ProteusException {
+//            var code = ProteusException.Code.UNKNOWN_ERROR
+//            if (proteusLastErrorCode != null) {
+//                code = ProteusException.fromProteusCode(proteusLastErrorCode)
+//            }
+//
+//            return ProteusException(err.message, code, err.cause)
+//        }
+//    }
+//}
```

Comment:
Any reason to keep a whole bunch of dead code here? Seems like we could just remove it.